codechecker_lib/analyzers/analyzer_types.py

```diff
@@ -173,6 +174,12 @@ def initialize_checkers(config_handler,
             else:
                 config_handler.disable_checker(checker_name)
 
+def __replace_env_var(matchobj):
+    env_var = matchobj.group(1)
+    if matchobj.group(1) not in os.environ:
+        LOG.error(env_var + ' environment variable not found')
```

Comment:
Could we provide a bit more specific error message to the users, that the problem is in the provided sa or tidy configuraton file.